frameit/lib/frameit/screenshot.rb

```diff
@@ -47,6 +47,8 @@ def device_name
         return 'iPad Air 2'
       when sizes::IOS_IPAD_PRO
         return 'iPad Pro'
+      when sizes::IOS_IPAD_PRO_12_9
+        return 'iPad Pro 3rd Generation'
```

Comment:
Can we change this "filename pattern"?
```suggestion
        return 'iPad Pro (12.9-inch) (3rd generation)'
```
This is what we require the screenshots to contain, so not introducing another string here would be nice. (This is of course still messy and inconsistent, but that's Apple's fault 🤷‍♀ )

Reply:
Thank you.
Yea, I'm cool with that.
I am using this string to locate the correct frame file. Those files do not currently have this same naming convention (with screen size and generation information).
I am happy to move toward this naming convention if it is appropriate.
What are your thoughts? @janpio

Reply:
Yeah, this is trying to be similar to https://github.com/fastlane/fastlane/blob/0de811cec8bc50e266b2724d96a3a3852c845e6e/deliver/lib/deliver/app_screenshot.rb#L140-L141

Reply:
Great 👍
I've tested the proposed change and it works, in tandem with the changes in the `frameit` repo of course.
I'll push both changes now.